src/root/rootobject.d

```diff
@@ -37,6 +38,7 @@ alias DYNCAST_IDENTIFIER = DYNCAST.identifier;
 alias DYNCAST_TUPLE = DYNCAST.tuple;
 alias DYNCAST_PARAMETER = DYNCAST.parameter;
 alias DYNCAST_STATEMENT = DYNCAST.statement;
+alias DYNCAST_CONDITION = DYNCAST.condition;
```

Comment:
These aliases are detritus from the D conversion. Use `DYNCAST.condition` instead.